httpserver.py

```diff
@@ -1,17 +1,25 @@
 from http.server import HTTPServer
-
-from py_server.config.web import get_http_config
-from py_server.reqhandler import ReqHandler
+# I like this. As all my comments suggest, use more of this bespoke design. 
+from config.web import get_http_config
+# // Request handler should be a 'factory' 
+# not something the HTTPSERVER module needs, let the factory look after that.
+# That way if things change, you only need to change your factory. 
+from reqhandler import ReqHandler
 
 if __name__ == "__main__":
 	(hostName, serverPort) = get_http_config()
+
 	webServer = HTTPServer((hostName, serverPort), ReqHandler)
 	print("Server started http://%s:%s" % (hostName, serverPort))
 
+	# Try catch is expensive, are you looking for something important here ?
+	# Ahh.. Python stuff - need a loop to focus it. Maybe put this into 'dev' mode ?
+	# // as for prod, no idea myself, you know better.
 	try:
```

Comment:
`Try catch is expensive` Jesus Adam, try catch is expensive to be used as a conditional 😉 

This try catch here is used to catch `ctl + c` from the user, this way we close the socket which is using a SO port